.github/scripts/dapr_bot.js

```diff
@@ -87,7 +88,8 @@ async function handleIssueCommentCreate({ github, context }) {
             await cmdOkToTest(github, issue, isFromPulls);
             break;
         case "/ok-to-perf":
-            await cmdOkToPerf(github, issue, isFromPulls);
+            const [_, ...commandArgs] = commandParts
+            await cmdOkToPerf(github, issue, isFromPulls, commandArgs.length > 0 ? commandArgs.join(" ") : undefined);
```

Comment:
```suggestion
            await cmdOkToPerf(github, issue, isFromPulls, commandParts.join(" "));
```

(This works if you use `shift()` above which removes the first element from the array)